modules/imaging_uploader/README.md

```diff
@@ -1,9 +1,73 @@
-To enable the Imaging Uploader to handle large files, please update the apache configuration file
-with the following values.
+# Imaging Uploader
 
-File to update : `/etc/php5/apache2/php.ini`
+## Purpose
 
-Sample values: 
+The imaging uploader is intended to allow users to upload, browse, and track 
+pipeline insertion progress.
+
+
+## Intended Users
+
+The primary users are MRI technicians or site coordinators to
+upload imaging scans for registered LORIS candidates and timepoints.
+
+## Scope
+
+The imaging uploader has the following built-in capabilities to facilitate 
+timely scan insertion into the LORIS database. Specifically, it allows to browse
+uploaded scans using the `Browse` tab, upload imaging scans using the `Upload` 
+tab, and track scan insertion status through the `Progress` column and a 
+`Log Viewer` which displays in either `Summary` or `Detailed` mode relevant 
+feedback messages about insertion success or failure causes.
+
+Uploaded scans can be overwritten if insertion status is `Not Started` or 
+`Failed` (i.e. if the `Progress` is neither `In Progress` nor `Success`). 
+
+
+NOT in scope:
+
+The imaging uploader does NOT read the DICOM files within the uploaded scans. 
+As such, it does not check if the files within the uploaded scan are of the 
+expected DICOM type, nor whether the  PatientName and/or PatientID DICOM headers 
+are properly de-identified according to the LORIS convention. This check is 
+however done as the first step on the LORIS-MRI side; i.e. as soon as the 
+insertion pipeline is triggered.
+
+## Imaging Uploader Requirements
+
+For a successful upload:
+- The uploaded file is expected to be of one of the following types: 
+`.tgz`, `.tar.gz` or `.zip`.
+- The filename should follow the:
+`PSCID_CandID_VisitLabel_OptionalSuffix` naming convention
+- It is expected that the candidate and visit are already created in the 
+database.
+
+
+## Permissions
+
+#### Module Permission
+
+The imaging uploader module uses one permission called `imaging_uploader` that 
+is necessary to have access to the module and gives the user the ability to 
+upload and browse all scans uploaded to the database.
+
+#### Filesystem Permission
+
+The path on the filesystem where the uploaded file go 
+(see section [Database Configuration](#Database Configurations)) should be 
```

Comment:
can I link to a later section in .md?

Reply:
Yes. You need to create an anchor at the location you want to link to.
ex: `<a name="database_config_link"></a>`

Then replace `(#Database Configurations)` by `(#database_config_link)`

Reply:
@xlecours 
thanks, it works!